### DISCUSSION TITLE
Just a conversion test to Discussions

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#2093] [General]
Just testing out how we can convet an issue to a discussion in the discussions section.

**Comment by ReinerBforartists** [comment]
Works like charm. But there is no way back it seems. Hm. So better be careful with converting issues to discussions ...

**Comment by Draise14** [comment]
Does the issue close when turned to a discussion?

**Comment by ReinerBforartists** [comment]
The issue vanishes from what i can see. There is no open or closed status in the discussions :)
